packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -118,6 +118,7 @@ export class ResourceService {
    * This function should be called from one of the other "Create[ResourceType] functions like CreateService, CreateMessageBroker etc."
    */
   private async createResource(
+    user: User,
```

Comment:
Are you sure that adding this new positional parameter won't break other calls?
Check any other reference of this method to be sure